### DISCUSSION TITLE
retries aware activities

### DISCUSSION THREAD

**Discussion by lyuboxa** [cschleiden/go-workflows#319] [Q&A]
Hello,

I'm wondering if there is a good way to provide the current number of activity retries an activity has undergone.
The use case for this is to determine at which part to force a behaviour rather than wait for it to resolve, e.g.
attempting to stop a service but if after the N retried the service is not stopped, attempt a force stop rather than timeout the workflow.

Thanks!
Lyubo

**Comment by cschleiden** [comment]
That seems like a good enhancement!
